terraform/cp4ba/main.tf

```diff
@@ -1,32 +1,37 @@
 # Provider block
 provider "ibm" {
   region           = var.region
-  version          = "~> 1.12"
   ibmcloud_api_key = var.ibmcloud_api_key
 }
 
+
 # Getting the OpenShift cluster configuration
 data "ibm_resource_group" "group" {
-  name = var.resource_group
+  name = var.resource_group_name
+}
+
+resource "null_resource" "mkdir_kubeconfig_dir" {
+  triggers  = { always_run = timestamp() }
+  provisioner "local-exec" {
+    command = "mkdir -p ${local.cluster_config_path}"
+  }
 }
 
 module "cluster" {
-  source = "git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak/tree/main/modules/roks"
-  enable = local.enable_cluster
-  on_vpc = var.on_vpc
+  source               = "git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak/tree/main/modules/roks"
+  enable               = local.enable_cluster
+  on_vpc               = var.on_vpc
 
-  project_name             = var.roks_project_name
-  owner                    = var.entitled_registry_user_email
-  environment              = var.environment
+  project_name         = var.cp4ba_project_name
```

Comment:
Why did you change this?  This information is retrieved by the installer script and typically set by the user. This cluster may not just be used for this cloud pak, so this shouldn't necessarily be the cp4ba_project_name. See https://github.com/ibm-hcbt/cloud-pak-sandboxes/blob/joel_cp4ba_sandbox/terraform/cp4i/main.tf for an example.